build.gradle.kts

```diff
@@ -237,7 +238,13 @@ subprojects {
                     inputs.property("moduleName", moduleName)
 
                     manifest {
-                        attributes("Automatic-Module-Name" to moduleName)
+                        attributes(
+                                "Automatic-Module-Name" to moduleName,
+                                "Built-By" to System.getProperty("user.name"),
+                                "Built-JDK" to System.getProperty("java.version"),
+                                "Implementation-Title" to project.name,
+                                "Implementation-Version" to project.version,
+                                "Release-Version" to releaseVersion)
```

Comment:
I'm unable to find any references to this attribute. Do you have a reference to a doc that describes what this is supposed to represent?

Reply:
releaseVersion? It's set to 8 (constant for the build).

Reply:
but is this a standard jar-manifest attribute? I don't think we should use manifest attributes that aren't pretty standard.

Reply:
See also https://docs.oracle.com/javase/9/docs/specs/jar/jar.html#main_attributesmain-attributes

Reply:
Previously (prior to kotlin migration we had: 
```
Source-Compatibility: 1.8
Target-Compatibility: 1.8
```
reflecting actual gradle build settings.
However currently we use javac `--release-version` which takes precedence (overwrites) source/target-compatibility settings (if any). Therefore I decide to replace the old attributes with the new one, reflecting current build settings (to avoid any confusion). 

If we want to stay aligned with the manifest spec, we shall get rid of this completely - but I believe this does bring additional value :)

Reply:
I don't know. If we invented these on our own, then I doubt any tool will read them. If you can find a reference to this attribute somewhere (specifications, documentation, 3rd party source code) we can keep them, otherwise I don't think it makes any sense.

Reply:
Done :)